www/running.inc

```diff
@@ -54,9 +54,9 @@
                     echo "URL: $url<br>\n";
                 } else {
                     if ($test['testinfo']['type'] === "traceroute") {
-                        echo "Host/IP: <a rel=\"nofollow\" href=\"http://$url\" target=\"_blank\">$url</a><br>\n";
+                        echo "Host/IP: <a class=\"urlLinkWrap\" rel=\"nofollow\" href=\"http://$url\" target=\"_blank\">$url</a><br>\n";
                     } else {
-                        echo "URL: <a rel=\"nofollow\" href=\"$url\" target=\"_blank\">$url</a><br>\n";
+                        echo "URL: <a class=\"urlLinkWrap\" rel=\"nofollow\" href=\"$url\" target=\"_blank\">$url</a><br>\n";
```

Comment:
This is hard to read and error prone, prefer single quote if possible

Reply:
Although it is easier to read, we should stay consistent with the rest of file.